boards/nucleo-wl55jc/include/periph_conf.h

```diff
@@ -149,6 +149,47 @@ static const i2c_conf_t i2c_config[] = {
 #define I2C_NUMOF           ARRAY_SIZE(i2c_config)
 /** @} */
 
+/**
+ * @name   ADC configuration
+ *
+ * Note that we do not configure all ADC channels,
+ * and not in the STM32WL55JC order. Instead, we
+ * just define 6 ADC channels, for the Nucleo
+ * Arduino header pins A0-A5 and the internal VBAT channel.
+ *
+ * To find appropriate device and channel find in the
+ * board manual, table showing pin assignments and
+ * information about ADC - a text similar to ADC1_IN[Y],
+ * where:
+ * [Y] - describes used channel - indexed from 1,
+ * for example ADC1_IN10 is channel 10
+ *
+ * For Nucleo-WL55JC this information is in board manual:
+ * Table 17, page 35.
```

Comment:
```suggestion
 * For Nucleo-WL55JC this information is in the board manual:
 * Table 17.
```

I would leave out the page numbers, because that can change with every revision of the document. The sections are (hopefully) more consistent.